### PR TITLE
NO-REF updated readme to clarify publish steps

### PR DESCRIPTION
**Pull request by hokei**
[Project Reno Jira Ticket](http://jira.nypl.org/browse/RENO-XXX)

[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)

## **This PR does the following:**
-

### Front End Review:
- [ ] View [the example in Storybook](https://reno-XXX-nypl.pantheonsite.io/themes/custom/nypl_emulsify/pattern-lab/public)
- [ ] Check against the [Metronome documentation](http://themetronome.co/components/xxx/?fresh=true)

### CODE REVIEW

**Comment by hokei** [comment]
please help me words.

File: README.md [hunk 1]
```diff
@@ -36,13 +36,12 @@ By default, the repo links everything under `src` for local development.  Someti
 
 ## Publishing
 You can publish npm modules from this repo by running:
-```lerna publish```
+```lerna publish``` from the main folder.
 Publishing from sub-folders using `npm publish` will result in broken components - don't do this!
-Please only use `custom version` and do the bump manually.
 
-`storybook` repo is private - Its version will be bumped, but it will not be published
+If you have bumped `package.json` as a part of your work, please use `custom version` and release the version that is in your `package.json`.  If you have not yet bumped `package.json` because you made no changes, but lerna asks you to bump your package, please apply a patch bump.  (It is likely that `package-lock.json` was changed.)
```
Comment (helenvholmes) [inline README.md]: If you have already bumped `package.json`:
1. Select the `custom version` option
2. Re-type the version you've specified in your `package.json`
3. ALWAYS bump the `storybook` package with a patch.

If you have NOT already bumped `package.json`:
1. Select `patch` option
2. ALWAYS bump the `storybook` package with a patch.

(Bullets are often easier to read in technical docs.)

**Review by helenvholmes — CHANGES_REQUESTED** [review]
Content's good, typed suggestion on the formatting in the comments

File: README.md (hunk 1 above)
Comment (hokei) [reply README.md]: sounds good!

**Review by helenvholmes — APPROVED** [review]

**Pull request merged**